TerminalDocs/tutorials/shell-integration.md

```diff
@@ -58,33 +58,28 @@ Supporting these features requires cooperation between your shell and the Termin
 To enable these features in the Terminal, you'll want to add the following to your settings:
 
 ```json
-"profiles":
-{
-    "defaults":
-    {
-        // Enable marks on the scrollbar
-        "showMarksOnScrollbar": true,
-
-        // Needed for both pwsh, CMD and bash shell integration
-        "autoMarkPrompts": true,
-
-        // Add support for a right-click context menu
-        // You can also just bind the `showContextMenu` action
-        "experimental.rightClickContextMenu": true,
-    },
-}
-"actions":
-[
-    // Scroll between prompts
-    { "keys": "ctrl+up",   "command": { "action": "scrollToMark", "direction": "previous" }, },
-    { "keys": "ctrl+down", "command": { "action": "scrollToMark", "direction": "next" }, },
-
-    // Add the ability to select a whole command (or its output)
-    { "command": { "action": "selectOutput", "direction": "prev" }, },
```

Comment:
so, this one's actually our bad! These commands definitely do exist if bound, they just aren't in the actions list by default.

Reply:
Gotcha I mis-assumed since my editor was complaining about them being invalid values, I've added them back into the docs. Is it worth making a PR to add them into the schema [json](https://github.com/microsoft/terminal/blob/main/doc/cascadia/profiles.schema.json) or is there a reason they've been left out in the meantime?